packages/sources/cryptocompare-test/src/endpoint/index.ts

```diff
@@ -0,0 +1,2 @@
+export { endpoint as cryptoWsEndpoint } from './crypto-ws'
+export { endpoint as cryptoEndpoint } from './crypto'
```

Comment:
Imma be annoying here too, to see if we can kill two birds with one stone: let's add a RoutingTransport that takes both of these, and turns it into one endpoint that chooses where to send the request based on the endpoint name ('crypto' would go to the batch one, 'crypto-ws' to the websocket one). This way we can start testing this new functionality with NOPs 😄